Scripts/Editor/Copiers/LegacyCopier.cs

```diff
@@ -369,6 +395,28 @@ internal static void CopyAllColliders(GameObject from, GameObject to, bool creat
                         ComponentUtility.CopyComponent(cFromArr[i]);
                         ComponentUtility.PasteComponentAsNew(cToObj);
 
+                        if (adjustScale)
+                        {
+                            float mul = Helpers.GetScaleMultiplier(cFromArr[i].transform, cToObj.transform);
+                            if (cToObj.TryGetComponent(out SphereCollider sphere))
```

Comment:
What is TryGetComponent? Is this extension from the VRChat SDK?
The tool is supposed to be independent from the sdk so don't use it.

Try something like this instead:

```cs
Collider col = cToObj.GetComponent<Collider>();
if(col is BoxCollider box)
{
	box.center *= mul;
	box.size *= mul;
}
```

Reply:
https://docs.unity3d.com/2019.4/Documentation/ScriptReference/Component.TryGetComponent.html is not from the VRChat SDK

Reply:
```
    public unsafe bool TryGetComponent<T>(out T component)
    {
      CastHelper<T> castHelper = new CastHelper<T>();
      this.TryGetComponentFastPath(typeof (T), new IntPtr((void*) &castHelper.onePointerFurtherThanT));
      component = castHelper.t;
      return (object) castHelper.t != null;
    }
```

Reply:
Ah so it's new in 2019.4. Cool